### ISSUE TITLE
Deprecated Node Runtime

### ISSUE THREAD

**Issue by atduarte** [stefanzweifel/sidecar-browsershot#83]
> We are contacting you as we have identified that your AWS Account currently has one or more Lambda functions using the Node.js 14 runtime.
>
> We are ending support for Node.js 14 in AWS Lambda. This follows Node.js 14 End-Of-Life (EOL) reached on April 30, 2023 [1].
>
> As described in the Lambda runtime support policy [2], end of support for language runtimes in Lambda happens in two stages. Starting November 27, 2023, Lambda will no longer apply security patches and other updates to the Node.js 14 runtime used by Lambda functions, and functions using Node.js 14 will no longer be eligible for technical support. In addition, you will no longer be able to create new Lambda functions using the Node.js 14 runtime. Starting January 25, 2024, you will no longer be able to update existing functions using the Node.js 14 runtime.
>
> We recommend that you upgrade your existing Node.js 14 functions to Node.js 18 before November 27, 2023.

**Comment by bobwurtz** [comment]
I received the same warning. If I'm reading this correctly, does this mean that starting on November 27, 2023 we won't be able to deploy new BrowsershotLambda functions? And does this mean that if I need to use this package in a new project that I need to deploy it before November 23, 2023? If it is already deployed, will it continue to work as expected into 2024?

Sorry if these questions are answered in the message posted above - I don't really understand how any of this works I just know that this package is amazing @stefanzweifel. Appreciate you putting this out there!

**Comment by stefanzweifel** [comment]
@bobwurtz I understood the text the same way. Starting November 27th 2023 you won't be able to deploy the current existing Lambda function to AWS. 

Starting January 25th 2024, you won't be able to update existing deployed functions.

---

I already spent several hours this week to figure out how to 

- upgrade the package to use Node 16 or Node 18
- update the used Chromium version to the latest version

I'm currently running into issues, to actually boot the new Chromium version on AWS. (Will probably open a PR with my work in progress in the next few days)

I can't promise anything, but I hope I can publish a new version of sidecar-browsershot before November 2023. 
I use this package in my own [SaaS-product](https://3.screeenly.com), so I kinda need a solution anyway. :)

**Comment by FlyLogik** [comment]
Thanks for the update!

**Comment by bobwurtz** [comment]
Thanks for the update!

**Comment by stefanzweifel** [comment]
I've just released v1.10.0 which adds support for Node 18.

In #85 the underlying layer has been updated to use the latest Chrome version (v116). I've also created our own [sidecar-browsershot-layer](https://github.com/stefanzweifel/sidecar-browsershot-layer) which contains a compatible `puppeteer-core` version.

I haven't stress tested the Chromium layer in my own apps yet, but I've invested ~15-20 hours in this PR and used the packages test suite to ensure everything works as expected.
If you run any into issues that are related to `sidecar-browsershot`, feel free to open issues.
